Tools/tsan/suppressions_free_threading.txt

```diff
@@ -41,6 +41,8 @@ race_top:tstate_is_freed
 race_top:type_modified_unlocked
 race_top:write_thread_id
 race_top:PyThreadState_Clear
+# see: https://github.com/python/cpython/issues/117721
+race_top:lock_PyThread_release_lock
```

Comment:
@colesbury found that there is a possible issue from https://github.com/python/cpython/blob/5e7eba09bcdafe361b491b3f8cf30d7dd2df0a78/Lib/test/lock_tests.py#L117-L121 which cause this TSAN error. We should fix it with a separate PR.